packages/react-native-payments-example/src/app/app.tsx

```diff
@@ -56,6 +58,21 @@ export const App = (): JSX.Element => {
         setTimeout(() => void paymentRequest.abort(), 1000);
     };
 
+    const handlePayWithoutShipping = (): void => {
```

Comment:
Maybe lets improve the example where you can set new fields with toggles for each type of the requested data: 
- requestBillingAddress
- requestPayerEmail
- requestPayerName
- requestPayerPhone
- requestShipping

and then we do not need separate methods/buttons?